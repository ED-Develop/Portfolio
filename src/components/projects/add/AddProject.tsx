import React from 'react';
import style from '../Projects.module.scss';
import {NavLink} from 'react-router-dom';
import {url} from '../../../utils/routeManager';
import {Button} from 'antd';
import {PlusOutlined} from '@ant-design/icons/lib';

export const AddProject = () => {
    return (
        <div className={style.projects__add}>
            <NavLink to={url('projects:add')}>
                <Button type='primary'><PlusOutlined/> Add Project</Button>
            </NavLink>
        </div>
    )
};