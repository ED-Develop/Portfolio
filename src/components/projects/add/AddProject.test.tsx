import React from 'react';
import {render} from '@testing-library/react';
import {BrowserRouter} from 'react-router-dom';
import {AddProject} from './AddProject';

describe('AddProject component', () => {
    test('Should renders correctly', () => {
        const {container} = render(
            <BrowserRouter>
                <AddProject/>
            </BrowserRouter>
        )

        expect(container).toMatchSnapshot();
    });
});