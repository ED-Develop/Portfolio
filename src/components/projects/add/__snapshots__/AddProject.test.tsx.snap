// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddProject component Should renders correctly 1`] = `
<div>
  <div
    class="projects__add"
  >
    <a
      href="/projects/add"
    >
      <button
        class="ant-btn ant-btn-primary"
        type="button"
      >
        <span
          aria-label="plus"
          class="anticon anticon-plus"
          role="img"
        >
          <svg
            aria-hidden="true"
            class=""
            data-icon="plus"
            fill="currentColor"
            focusable="false"
            height="1em"
            viewBox="64 64 896 896"
            width="1em"
          >
            <defs>
              <style />
            </defs>
            <path
              d="M482 152h60q8 0 8 8v704q0 8-8 8h-60q-8 0-8-8V160q0-8 8-8z"
            />
            <path
              d="M176 474h672q8 0 8 8v60q0 8-8 8H176q-8 0-8-8v-60q0-8 8-8z"
            />
          </svg>
        </span>
        <span>
           Add Project
        </span>
      </button>
    </a>
  </div>
</div>
`;
